### PR TITLE
Feature/tracking

### PR DESCRIPTION
**Pull request by nemoeslovo**
addressing issue #2. Implemented logging to es

image

### CODE REVIEW

File: generic-update-script.rb [hunk 1]
```diff
@@ -131,13 +139,20 @@
...
   #########################################
-    checker = Dependabot::UpdateCheckers.for_package_manager(package_manager).new(
+  checker = Dependabot::UpdateCheckers.for_package_manager(package_manager).new(
     dependency: dep,
     dependency_files: files,
     credentials: credentials,
   )
-  next unless dep.display_name.match(/^Agoda/) 
-  next if checker.up_to_date?
+  unless dep.display_name.match(/^Agoda/) then
```
Comment (dicko2) [inline generic-update-script.rb:147]: #1 we need to change this to address this issue, will there be anohter PR for this?

Comment (nemoeslovo) [reply generic-update-script.rb:147]: yep, separate issue

Comment (nemoeslovo) [reply generic-update-script.rb]: pushed within the same pr to avoid conflicts

**Comment by nemoeslovo** [comment]
@dicko2 I updated PR with  #1  issue addressed

**Pull request merged**